### PR TITLE
19521 added output bugs fixes

### PR DESCRIPTION
**Pull request by PaulGarewal**
*Issue #:* /bcgov/entity#19521

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/report-templates/certificateOfAmalgamation.html [hunk 1]
```diff
@@ -34,15 +34,15 @@
                         <tr class="no-page-break">
                             <td class="col-50">
                                 <div class="{{'upper-text' if entity.legalName!='Not Available'}}">
-                                    and {{ entity.name }} ({{ entity.identifier }})
+                                    and {{ entity.legalName }} ({{ entity.identifier }})
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/certificateOfAmalgamation.html:37]: this will fail now. it only work for foreign business

Comment (vysakh-menon-aot) [reply legal-api/report-templates/certificateOfAmalgamation.html:37]: You can include this change. i will fix it in my pr
image

File: legal-api/report-templates/template-parts/amalgamation/amalgamatingCorp.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
         <tr class="no-page-break">
         <td class="col-50">
             <div class="{{'upper-text' if entity.legalName!='Not Available'}} pt-2">
-                {{ entity.name }}
+                {{ entity.legalName }}
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/template-parts/amalgamation/amalgamatingCorp.html:19]: same as above

File: legal-api/report-templates/certificateOfAmalgamation.html [hunk 3]
```diff
@@ -34,15 +34,15 @@
...
                                 </div>
                             </td>   
                             </tr>
                         {%else%}
                         <tr class="no-page-break">
                         <td class="col-50">
                             <div class="{{'upper-text' if entity.legalName!='Not Available'}}">
-                                {{ entity.name }}
+                                {{ entity.legalName }}
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/certificateOfAmalgamation.html:45]: same as above

File: legal-api/report-templates/template-parts/amalgamation/amalgamatingCorp.html [hunk 4]
```diff
@@ -16,7 +16,11 @@
         <tr class="no-page-break">
         <td class="col-50">
             <div class="{{'upper-text' if entity.legalName!='Not Available'}} pt-2">
-                {{ entity.name }}
+                {% if entity.legalName %}
+                    {{ entity.legalName }}
+                {%else%}
+                    {{ entity.legalName }}
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/template-parts/amalgamation/amalgamatingCorp.html:22]: {{ entity.name }}

File: legal-api/report-templates/certificateOfAmalgamation.html [hunk 5]
```diff
@@ -34,16 +34,25 @@
...
                         {%else%}
                         <tr class="no-page-break">
                         <td class="col-50">
                             <div class="{{'upper-text' if entity.legalName!='Not Available'}}">
-                                {{ entity.name }}
+                                {% if entity.legalName %}
+                                {{ entity.legalName }}
                                 ({{ entity.identifier }}),
+                                {%else%}
+                                {{ entity.legalName }}
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/certificateOfAmalgamation.html:53]: should be `{{ entity.name }}`

File: legal-api/report-templates/template-parts/amalgamation/amalgamatingCorp.html (hunk 4 above)
Comment (PaulGarewal) [reply legal-api/report-templates/template-parts/amalgamation/amalgamatingCorp.html]: Great catch

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**